### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by dskatz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
-# cdnvalidator
+# cdnvalidator service
+
+[![Build Status](https://drone.corp.mongodb.com/api/badges/kanopy-platform/cdnvalidator/status.svg)](https://drone.corp.mongodb.com/kanopy-platform/cdnvalidator)
+
+The cdnvaildator provides multi-tenant entitlement abstraction layer into CloudFront invalidations which may have different owners, performance, and cache control requirements.
```
Comment (yuzhouliu9) [inline README.md:5]: nit: spelling, and I think this flows a bit better to breaks up the terse sentence:

```suggestion
The cdnvalidator provides an abstraction layer for CloudFront invalidations. It provides control for performance, different owners of distribution paths, and cache control requirements.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+
+## Architecture
+
+The CDN Vaildator provides a RESTful API detailed in the [OpenAPI documentation](./swagger/swagger.json).
```
Comment (yuzhouliu9) [inline README.md:9]: spelling: `Validator`

File: README.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+
+### Authn/Authz
+
+The service relies on being deployed behind a reverse proxy that handles forward authentication and validation of the JWT.  Authorization decisions will be made against the `groups` and `scps` claims of a JWT.
```
Comment (yuzhouliu9) [inline README.md:13]: nit: is the literal `scp` instead?

**Review by colinhoglund — COMMENTED** [review]
Same comments at Yuzhou

File: README.md (hunk 1 above)
Comment (dskatz) [reply README.md]: I took most of this, but changed some of it.

**Review by yuzhouliu9 — APPROVED** [review]

**Pull request merged**